sass/pages/_careers.scss

```diff
@@ -0,0 +1,126 @@
+.team {
+  height: 650px;
+  overflow: hidden;
+  position: relative;
+  width: 100%;
+}
+
+.team-image {
+  background: url('../img/team.jpg');
+  background-position: center;
+  background-size: cover;
+  height: 100%;
+  overflow: hidden;
+  width: 100%;
+}
+
+.team-overlay {
+  background-color: $overlay-color;
+  bottom: 0;
+  position: absolute;
+  top: 0;
+  width: 100%;
+}
+
+.team-center {
+  left: 50%;
+  position: absolute;
+  text-align: center;
+  top: 50%;
+  transform: translateX(-50%)translateY(-50%);
+  z-index: 20;
+}
+
+.team-caption {
+  color: $white-color;
+  font-size: $font-40;
+  letter-spacing: .4px;
+  text-transform: uppercase;
+  word-spacing: 5px;
+}
+
+.team-button {
```

Comment:
@priyanka0901 Why are you writing your own CSS for the button if it already available in the library.